src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -44,43 +48,57 @@ public class DsStorage implements AutoCloseable {
     private static final String CTIME_COLUMN = "ctime";
     private static final String MTIME_COLUMN = "mtime";
     private static final String PARENT_ID_COLUMN = "parentid";
-    private static final String KALTURA_REFERENCE_ID_COLUMN = "kalturareferenceid";
-    private static final String KALTURA_INTERNAL_ID_COLUMN = "kalturainternalid";
-    
-    
+    private static final String REFERENCE_ID_COLUMN = "referenceid";
+    private static final String KALTURA_ID_COLUMN = "kalturaid";
+    private static final String MAPPING_REFERENCE_ID_COLUMN = "referenceid";
+    private static final String MAPPING_KALTURA_ID_COLUMN = "kalturaid";
```

Comment:
Might be me not seeing the benefit here, but we are we declaring two pairs of identical variables here? Is it to be able to tell the tables apart?

Reply:
Renamed so prefix for both. It is different tables but column have same name, so I prefer two different variables